python/QC4_HV.py

```diff
@@ -21,7 +21,7 @@
 cmd.append("--SelectRowEnd= 37")
 cmd.append("--CanvRangeX= 0,1000")
 cmd.append("--CanvRangeY= 0,7")
-cmd.append("--YaxisScale= True")
+cmd.append("--YaxisScale=true")
```

Comment:
The argparse module improves on the standard library optparse module in a number of ways including:
-handling positional arguments
-supporting sub-commands
-allowing alternative option prefixes like + and /
-handling zero-or-more and one-or-more style arguments
-producing more informative usage messages
-providing a much simpler interface for custom types and actions

I just thought that since argparse is just an improvement of optparse will be better to use